docs/source/quantization.md

```diff
@@ -0,0 +1,245 @@
+| ONNX Runtime | [MLAS](https://github.com/microsoft/onnxruntime/tree/master/onnxruntime/core/mlas) | Weight (int8) | Activation (uint8) |
+
+
+#### Quantization Scheme
++ Symmetric Quantization
+    + int8: scale = 2 * max(abs(rmin), abs(rmax)) / (max(int8) - min(int8) - 1)
++ Asymmetric Quantization
+    + uint8: scale = (rmax - rmin) / (max(uint8) - min(uint8)); zero_point = min(uint8)  - round(rmin / scale) 
+
+#### Reference
++ MLAS:  [MLAS Quantization](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/onnx_quantizer.py) 
+
+### Quantization Approaches
+
+Quantization has three different approaches:
+1) post training dynamic quantization
+2) post training static  quantization
+
+The first two approaches belong to optimization on inference. The last belongs to optimization during training. Currently. ONNX Runtime doesn't support the last one.
+
+#### Post Training Dynamic Quantization
+
+The weights of the neural network get quantized into int8 format from float32 format offline. The activations of the neural network is quantized as well with the min/max range collected during inference runtime.
+
+This approach is widely used in dynamic length neural networks, like NLP model.
+
+#### Post Training Static Quantization
+
+Compared with `post training dynamic quantization`, the min/max range in weights and activations are collected offline on a so-called `calibration` dataset. This dataset should be able to represent the data distribution of those unseen inference dataset. The `calibration` process runs on the original fp32 model and dumps out all the tensor distributions for `Scale` and `ZeroPoint` calculations. Usually preparing 100 samples are enough for calibration.
+
+This approach is major quantization approach people should try because it could provide the better performance comparing with `post training dynamic quantization`.
+
+## With or Without Accuracy Aware Tuning
+
+Accuracy aware tuning is one of unique features provided by Intel(R) Neural Compressor, compared with other 3rd party model compression tools. This feature can be used to solve accuracy loss pain points brought by applying low precision quantization and other lossy optimization methods. 
+
+This tuning algorithm creates a tuning space based on user-defined configurations, generates quantized graph, and evaluates the accuracy of this quantized graph. The optimal model will be yielded if the pre-defined accuracy goal is met.
+
+Neural compressor also support to quantize all quantizable ops without accuracy tuning, user can decide whether to tune the model accuracy or not. Please refer to "Get Start" below.
+
+### Working Flow
+
+Currently `accuracy aware tuning` only supports `post training quantization`.
+
+User could refer to below chart to understand the whole tuning flow.
+
+<img src="./imgs/workflow.png" width=600 height=480 alt="accuracy aware tuning working flow">
```

Comment:
![image](https://github.com/intel/neural-compressor/assets/51692656/5c46806b-6282-49dc-8f4c-9bbfeb14c639)
Check the scale